api/scpca_portal/readme_file.py

```diff
@@ -10,10 +11,13 @@
 TEMPLATE_FILE_PATH = TEMPLATE_ROOT / "readme.md"
 
 
-def get_file_contents(download_config: Dict, project) -> str:
+def get_file_contents(download_config: Dict, projects) -> str:
```

Comment:
```suggestion
def get_file_contents(download_config: Dict, projects: Iterable) -> str:
```

youll need to import Iterable

Reply:
See at 68053c986dc5b971a57bbee8039042a741dc20c8.